common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -99,14 +104,9 @@ public static void init(String modVersion, boolean isDevelopmentEnvironment) {
         ManagerRegistry.init();
     }
 
-    private static void initFeatures() {
-        // Init all features. Now resources (i.e I18n) are available.
-        FeatureRegistry.init();
-    }
-
     private static void parseVersion(String versionString) {
         if (isDevelopmentEnvironment()) {
-            LOGGER.info("Wynntils running on version " + versionString);
+            LOGGER.info("Wynntils running on version {}", versionString);
```

Comment:
If you are updating this, why not add the modloader as well? It is useful if we ever see parts of a log. Also, the message is really unclear. "running on version ..." sounds like we are giving the java version. But I assume this is the Wynntils version..?

So maybe something like `"Wynntils version {} (using {} on Minecraft {}) is now loaded", modVersion, modLoader, mcVersion` or however it needs to be to be correct..?